hypothesis-python/src/hypothesis/core.py

```diff
@@ -936,10 +1011,17 @@ def wrapped_test(*arguments, **kwargs):
                         "generated?"
                     )
 
+            # There was no @reproduce_failure, so start by running any explicit
+            # examples from @example decorators.
+
             execute_explicit_examples(
                 test_runner, test, wrapped_test, settings, arguments, kwargs
             )
 
+            # All of the explicit examples passed (if any), so the next step
```

Comment:
nit: The placement of the (if any) in this sentence is weird. Maybe "All of the explicit examples (if there were any) passed"?